<launch>
    <node name="create_zones" pkg="mir_navigation" type="create_zones_web.py" output="screen" />
    <node name="publish_zones" pkg="mir_navigation" type="publish_zones.py" output="screen" />
    <node name="check_zones" pkg="mir_navigation" type="check_zones.py" output="screen" />
    <node name="process_zones" pkg="mir_navigation" type="process_zones.py" output="screen" />
    <node name="visualize_zones" pkg="mir_navigation" type="visualize_zones.py" output="screen" />
</launch>
